packages/calypso-products/src/plans-list.tsx

```diff
@@ -2318,6 +2322,13 @@ const getPlanJetpackStarterDetails = (): IncompleteJetpackPlan => ( {
 	getInferiorFeatures: () => [ FEATURE_JETPACK_BACKUP_DAILY, FEATURE_JETPACK_BACKUP_DAILY_MONTHLY ],
 } );
 
+const getPlanJetpackGrowthDetails = (): IncompleteJetpackPlan => ( {
+	group: GROUP_JETPACK,
+	type: TYPE_JETPACK_GROWTH,
+	getTitle: () => translate( 'Growth', { context: 'Jetpack product name' } ),
+	getDescription: () => translate( 'Accelerate your growth.' ),
```

Comment:
Unless you've seen this tagline elsewhere, I'd go with something like `Grow your audience effortlessly` which is on our https://jetpack.com/growth landing page 😄

Reply:
Hot!